internal/api/github.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/google/go-github/v51/github"
+)
+
+func findRelease(matchFunc func(*github.RepositoryRelease) bool) (*github.RepositoryRelease, error) {
```

Comment:
ah sorry Achille, I know I mentioned we had to problems with getting latest using api but I recall now that was in the installation script, there seems to be a direct API call to get the latest release:

https://pkg.go.dev/github.com/google/go-github/v51/github#RepositoriesService.GetLatestRelease

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release